apps/emqx/include/asserts.hrl

```diff
@@ -29,3 +29,17 @@
         )
     )
 ).
+
+-define(assertInclude(Pattern, List),
+    ?assert(
+        lists:any(
+            fun(El) ->
```

Comment:
Additional minor remark: usually it's also smart to give any function variable introduced inside a macro some sort of ugly name like `El__` to avoid potential collisions with the outside code and `erlc` complaining about variable shadowing.